docs/multicluster/user-guide.md

```diff
@@ -559,6 +559,42 @@ In future releases, some additional tooling may become available to automate the
 creation of ResourceExports for ACNPs, and provide a user-friendly way to define
 Multi-cluster NetworkPolicies to be enforced in the ClusterSet.
 
+## Antrea Multicluster Pod to Pod Connectivity
+Since Antrea v1.9.0, Multi-cluster allows traffic between Pods in different clusters. If you want to enable this feature,
+the Pod CIDRs must be set in the ConfigMap `antrea-mc-controller-config` like below. You can edit the config in
+[antrea-multicluster-member.yml](../../multicluster/build/yamls/antrea-multicluster-member.yml) or using `kubectl edit`.
+
+```yaml
+apiVersion: v1
+data:
+  controller_manager_config.yaml: |
+    apiVersion: multicluster.crd.antrea.io/v1alpha1
+    kind: MultiClusterConfig
+    health:
+      healthProbeBindAddress: :8080
+    metrics:
+      bindAddress: "0"
+    webhook:
+      port: 9443
+    leaderElection:
+      leaderElect: false
+    serviceCIDR: ""
```

Comment:
Do we need users to provide "serviceCIDR"? @luolanzone

Reply:
@jianjuns , no, if it's empty, controller will try to detect it automatically.